test/loopback-dl.test.js

```diff
@@ -1911,3 +1911,61 @@ describe('ModelBuilder options.models', function() {
     assert.deepEqual(codes.number, ['unknown-property']);
   });
 });
+
+describe('updateOnly', function() {
+  it('verify forceId', function(done) {
```

Comment:
We are using BDD style, where the test names should read like an English sentence, e.g. `it verifies forceId`. They should say what is being tested (that's usually specified in `describe`), what is the observed behaviour and what were the conditions of the test.

In this case, I think a better test name is `it('sets forceId to true when model id is generated')`.

Feel free to use different test names as long as they follow the BDD style.